src/Core/Plugin.php

```diff
@@ -119,6 +120,13 @@ public static function load($plugin, array $config = [])
             return;
         }
 
+        if (!Configure::check('pluginPaths')) {
```

Comment:
Here you check for key `pluginPaths` but below `'plugins.' . $plugin` is used for reading.